project/dexter-vd-cpp/src/java/com/samsung/sec/dexter/vdcpp/util/CDTASTUtil.java

```diff
@@ -0,0 +1,401 @@
+		String name=functionNameExpression.toString();
+		
+		return (name.compareTo(functionName)==0);
+	}
+	
+	public static boolean checkDimensionOfPointer(IASTNode node, int dimension){		
+		IASTName name=null;
+		if(node instanceof IASTIdExpression){	
+			name =((IASTIdExpression) node).getName();
+		}else if(node instanceof IASTDeclarator){
+			name =((IASTDeclarator) node).getName();
+		}
+		
+		if(name==null)	return false; 								
+		final IBinding binding = name.resolveBinding();
+		if (binding == null)	
+			return false;
+		
+		if(!(binding instanceof IVariable))
+			return false;
+		
+		IType type=((IVariable)binding).getType();
+		if(!(type instanceof IPointerType))
+			return false;		
+	
+		String typeString=((IPointerType)type).toString();
+
+		return (getCountOfCharacter(typeString, '*')==dimension);
+	}
+	
+	public static IASTNode getAssignedTo(IASTNode assignedFrom){
+		IASTNode assignedTo = null;
+		
+		IASTNode parent =((IASTNode) assignedFrom).getParent();
+		while(isBracketedPrimaryUnaryExpression(parent))
+			parent =((IASTNode) parent).getParent();
+		
+		if(parent instanceof IASTCastExpression)
+			parent =((IASTNode) parent).getParent();
+					
+		if(parent instanceof IASTExpression){
+			if(isAssignBinaryExpression((IASTExpression)parent))	
+				assignedTo = ((IASTBinaryExpression) parent).getOperand1();
+		}
+		else if(parent instanceof IASTEqualsInitializer){
+			parent =((IASTNode) parent).getParent();								
+			
+			if(parent instanceof IASTDeclarator)
+				assignedTo = parent;
+		}
+		
+		return assignedTo;
+	}
+	
+	private static boolean isBracketedPrimaryUnaryExpression(IASTNode node){
+		return ((node instanceof IASTUnaryExpression) &&
+				(((IASTUnaryExpression) node).getOperator() == IASTUnaryExpression.op_bracketedPrimary));
+	}
+	
+	private static int getCountOfCharacter(String string, char character){
+		int count = 0;
+		for(int i =0; i < string.length(); i++)
+		    if(string.charAt(i) == character)
+		        count++;
+
+		return count;
+	}
+	
+	public static String getValueOfExpression(IASTExpression operandExpression){
+		String value="";
+		
+		if(operandExpression instanceof IASTLiteralExpression){
+			value=getValueOfLiteralExpression(operandExpression);		
+		}
+		/*else if(operandExpression instanceof IASTIdExpression){		
+			value=getValueOfIdExpression(operandExpression);
+		}*/
+		
+		return value;
+	}
+	
+	private static String getValueOfLiteralExpression(IASTExpression operandExpression){
+		return operandExpression.toString();
+	}
+	
+	private static String getValueOfIdExpression(IASTExpression expression) {
+		String value="";
+		
+		IValue val = null;
+		val.numericalValue();
+		
+		return value;				
+	}	
+	
+	public static Integer convertIntoDecimalNumber(String string) 
+	{				
+		Integer outputDecimal=0;	
+		
+		try
+		{	
+			string = string.replaceAll("(U|u|LL|ll|L|l)", "");		
+			
+			if(string.contains("x") ||string.contains("X") )		//ex) 0xff : hexa-decimal
```

Comment:
You can use charAt method instead of contains()
string.charAt(1) == 'x' || string.charAt(1) == 'X'